### ISSUE TITLE
[Task] Review and fix files declaration

### ISSUE THREAD

**Issue by JimiC** [robertohuertasm/vscode-icons#424]
**Checklist:**

- [x] I'm sure this issue is not a *duplicate*?

Currently, some file declarations are incorrect. (i.e. `.editorconfig` is a configuration file with no extension, and has been declared as a file extension).

**Task assignment:**
Go through all entries in `supportedExtensions.js` and correct the file declaration.

**Comment by JimiC** [comment]
This is just a placeholder issue for the PR I'm gonna submit (sometime until next weekend, I hope).

**Comment by JimiC** [comment]
@robertohuertasm 

I see some entries like 
```
    { icon: 'bower', extensions: ['bowerrc'], contribType: ctype.filename },
    { icon: 'bower', extensions: ['bower.json'], contribType: ctype.filename },
```
can they be concatenated to a one line declaration, like
``` { icon: 'bower', extensions: ['bowerrc', 'bower.json'], contribType: ctype.filename } ```
?

**Comment by JimiC** [comment]
@robertohuertasm I think I have answered my own question. I figured that if two filenames have a same icon and one filename has an extension, but still belongs to the `fileNames` group, it get on a new line.

**Comment by robertohuertasm** [comment]
@JimiC those entries should be merged into one. This is a legacy issue. We just removed the old custom funcionality and before we needed to separate the different filenames into different entries in order to generate our custom replacements. 

As this is now not needed at all, we may have several cases like this where we could merge several entries into one. If they all are `filenames` or `fileextensions` then can be declared as one entry.
